hist/hist/inc/TFormula.h

```diff
@@ -174,6 +184,7 @@ class TFormula : public TNamed
       kLambda        = BIT(17)     ///< Set to true if TFormula has been build with a lambda
    };
    using GradientStorage = std::vector<Double_t>;
+   using HessianStorage = std::vector<Double_t>;
```

Comment:
Is that really worth it? The extra level of indirection doesn't seem to help much - but makes it more difficult for users to figure out what that parameter type is.